From 7fef7374444f995c003e11ab3daab8283bd4fa30 Mon Sep 17 00:00:00 2001
From: Haikun Wang <Haikun.Wang@freescale.com>
Date: Fri, 10 Jul 2015 03:09:50 +0800
Subject: [PATCH 104/129] arm: dts: ls1021a: Add transfer mode property in
 DSPI2 node

DSPI new driver can select transfer mode(tcfq/eoq) to work.
The property will be read from dtsi node.
Add the property tcfq-mode for LS1021a.

Signed-off-by: Haikun Wang <haikun.wang@freescale.com>
Change-Id: I09efa9277364b79d075a1de94bd04111e2434576
Reviewed-on: http://git.am.freescale.net:8181/39515
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Zhengxiong Jin <Jason.Jin@freescale.com>
[Xulin:Original patch taken from
Freescale-Linux-SDK-for-LS1021A-IOT-Rev2-v0.4-SOURCE-20150907-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index 4a3563b..bc9d560 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -288,6 +288,7 @@
 			clocks = <&platform_clk 1>;
 			spi-num-chipselects = <5>;
 			big-endian;
+			tcfq-mode;
 			status = "disabled";
 		};
 
-- 
1.7.5.4

